src/audio/pipeline.c

```diff
@@ -327,18 +327,18 @@ int pipeline_comp_connect(struct pipeline *p, struct comp_dev *source_comp,
 	if (sink_buffer->source && sink_buffer->sink)
 		sink_buffer->connected = 1;
 
-	tracev_pipe_with_ids(p, "pipeline_comp_connect(), (source_comp->comp.id"
-			     " << 16) | sink_buffer->ipc_buffer.comp.id = %08x",
-			     (source_comp->comp.id << 16) |
-			     sink_buffer->ipc_buffer.comp.id);
+	tracev_pipe("pipeline_comp_connect(), (source_comp->comp.id"
```

Comment:
These messages would not be needed since all the info is conveyed above.

Reply:
I've removed them